ngi_pipeline/conductor/flowcell.py

```diff
@@ -411,20 +375,20 @@ def parse_casava_directory(fc_dir):
 
 def parse_xten_directory(fc_dir):
     """
-    Traverse a xten-generated directory structure and return a dictionary
+    Traverse a XTen-generated directory structure and return a dictionary
     of the elements it contains.
     The flowcell directory tree for HiSeq xtenruns has (roughly) the structure:
 
     ├── Demultiplexing
     │   ├── J_Lundeberg_14_24
-    │   │   ├── P1775_112
-    │   │   ├── P1775_120
-    │   │   ├── P1775_128
-    │   │   ├── P1775_136
-    │   │   ├── P1775_144
-    │   │   ├── P1775_152
-    │   │   ├── P1775_160
-    │   │   └── P1775_168
+    │   │   ├── Sample_P1775_112
```

Comment:
My bad!!! Sample is not necessary and I wanted to get rid of it but I pointed you to an example where sample was still there.
The Adults version of the flowcell "141229_ST-E00214_0019_AH00XXXXXX" is without "Sample" and it is like this that should be (as we agree with Patrick)